### PR TITLE
Tutorial Explorer: remove robotics page

### PR DESCRIPTION
**Pull request by breville**
Until now, we had a variant of the Tutorial Explorer at https://code.org/learn/robotics and https://hourofcode.com/learn/robotics which was the way to see tutorials that had an `activity_type` filter group tag of `robotics`.  

Now, each robotics page permanently redirects to its respective parent page, and the parent no longer has a button that goes to its robotics page.  Robotics tutorials are now shown by default, and  the `platform` filter group now has a `robotics` filter (labeled "Robotics, circuits").

This change won't be merged until the `cdo-tutorials` gsheet is updated with these new tags.

Screen Shot 2022-09-27 at 8 29 08 AM

### CODE REVIEW

File: apps/src/tutorialExplorer/backButton.jsx [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-    marginBottom: 13
-  }
-};
-
-const BackButton = props => (
-  <a href="/learn">
-    <button type="button" style={styles.backButton}>
-      <i className="fa fa-arrow-left" aria-hidden={true} />
-      &nbsp;
-      {i18n.backButtonBack()}
```
Comment (bencodeorg) [inline apps/src/tutorialExplorer/backButton.jsx:19]: Remove from translations?

**Review by bencodeorg — APPROVED** [review]
Nice cleanup! We're not auto-selecting the robotics checkbox on redirect, yeah? Too much work to do that?

File: apps/src/tutorialExplorer/backButton.jsx (hunk 1 above)
Comment (breville) [reply apps/src/tutorialExplorer/backButton.jsx:19]: Nice catch!

**Comment by breville** [comment]
> Nice cleanup! We're not auto-selecting the robotics checkbox on redirect, yeah? Too much work to do that?

Nice insight!  Interestingly, we do have a (little-used) [facility](https://github.com/code-dot-org/code-dot-org/blob/668f93416df0326c9f695cd27b335b0cd7267fbe/apps/src/tutorialExplorer/README.md#url-parameters) to pre-select filters with a URL.  That said, we weren't getting much traffic to those pages and it might be desirable to start with the default filters to keep things simple.

**Comment by bencodeorg** [comment]
> > Nice cleanup! We're not auto-selecting the robotics checkbox on redirect, yeah? Too much work to do that?
> 
> Nice insight! Interestingly, we do have a (little-used) [facility](https://github.com/code-dot-org/code-dot-org/blob/668f93416df0326c9f695cd27b335b0cd7267fbe/apps/src/tutorialExplorer/README.md#url-parameters) to pre-select filters with a URL. That said, we weren't getting much traffic to those pages and it might be desirable to start with the default filters to keep things simple.

Oh cool, good to know! Sounds good re: keeping simple.

**Pull request merged**